src/components/toJSONStream.js

```diff
@@ -0,0 +1,16 @@
+const parallelTransform = require('parallel-transform');
```

Comment:
Hi @Joxit, what's the reason for using `parallel-transform` here?

I had a quick look over the code and it doesn't seem to defer non-blocking I/O like the name implies?

Reply:
Hi, I thought that having the process in parallel would improve performance during the SQLite query. But you're right, it may not change anything.
I did not compare the two versions :confused:.

Reply:
I suspect this will either have no impact on performance or it'll slightly slow things down.

Node is single-threaded and so the only way to achieve 'parallelism' is to schedule tasks on to the next 'tick' of the event loop.
This only ever has a benefit for non-blocking I/O where the 'task' each cycle is simply checking if it's ready to return data, and then only using large amounts of CPU when there is something ready to do.

It's more like 'deferring work' on a single thread rather than true parallelism.

In this case, the task is `JSON.parse()` which is a CPU-bound blocking operation, so AFAIK there is no way it could benefit from any parallelism, if anything it'll slow things down due to the additional bookkeeping overhead ☹️ 

I'd be happy to be proven wrong, otherwise, I think we can just remove it and use a regular stream for simplicity sake 😸

Reply:
Yep, you were right ! I did some tests and this does not improve performance, now I use simple through2 :smile: 

Thx for this review ! :grin: